### PR TITLE
Formatting changes using the formatting tool 

### PR DESCRIPTION
**Pull request by jebbinSCL**
Formatting 
Import alphabetical reordering

### CODE REVIEW

File: TrelloMigrate/SrmApiClient.fs [hunk 1]
```diff
@@ -1,35 +1,25 @@
...
-        post endpoint session
+    let post (session : Session) = post endpoint session
 
 module Correspondence = 
     let private endpoint = Uri(commsServiceUri, "Correspondence")
-
-    let post (item : CorrespondenceItem) = 
-        post endpoint item
\ No newline at end of file
+    let post (item : CorrespondenceItem) = post endpoint item
```
Comment (pallenSL) [inline TrelloMigrate/SrmApiClient.fs]: Is it the formatting tool that deletes your last newline and gives us this warning character in every file?

Comment (jebbinSCL) [inline TrelloMigrate/SrmApiClient.fs]: Adjusted for all files

Comment (jebbinSCL) [inline TrelloMigrate/SrmApiClient.fs]: That was already there though.

**Comment by pallenSL** [comment]
I wish it was more consistent, e.g. indenting _all_ match clauses, but it's more good than bad.
:shipit:

**Pull request merged**